nms/app/packages/magmalte/server/network/routes.js

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow
+ * @format
+ */
+
+import type {ExpressResponse} from 'express';
+import type {FBCNMSRequest} from '@fbcnms/auth/access';
+import type {
+  network_cellular_configs,
+  network_dns_config,
+  tier,
+} from '@fbcnms/magma-api';
+
+import asyncHandler from '@fbcnms/util/asyncHandler';
+import express from 'express';
+
+import MagmaV1API from '../magma';
+import {AccessRoles} from '@fbcnms/auth/roles';
+import {CWF, FEG, FEG_LTE, LTE, SYMPHONY, XWFM} from '@fbcnms/types/network';
+import {access} from '@fbcnms/auth/access';
+import {difference} from 'lodash';
+
+const logger = require('@fbcnms/logging').getLogger(module);
+
+const router: express.Router<FBCNMSRequest, ExpressResponse> = express.Router();
+
+const DEFAULT_CELLULAR_CONFIG: network_cellular_configs = {
+  epc: {
+    cloud_subscriberdb_enabled: false,
+    default_rule_id: 'default_rule_1',
+    lte_auth_amf: 'gAA=',
+    lte_auth_op: 'EREREREREREREREREREREQ==',
+    mcc: '001',
+    mnc: '01',
+    network_services: ['policy_enforcement'],
+    relay_enabled: false,
```

Comment:
Modify this to remove `relay_enabled` and have `hss_relay_enabled` and `gx_gy_relay_enabled`, both set to `false` by default

Reply:
staging is still not updated with this change - https://api-staging.magma.etagecom.io/apidocs/v1/swagger.yml, can you push a version containing this change to staging. Thanks.

Reply:
@andreilee Sure. I will do this once staging is updated with these changes.